lib/PuppeteerSharp/JSCoverage.cs

```diff
@@ -1,7 +1,9 @@
 ﻿using PuppeteerSharp.Messaging;
 using System;
+using System.Linq;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using static PuppeteerSharp.Messaging.ProfilerTakePreciseCoverageResponse;
```

Comment:
is this needed?

Reply:
Yes because `ProfilerTakePreciseCoverageResponseRange` is a sub-class. I thought that making a new file for `ProfilerTakePreciseCoverageResponseRange` was "too much". What do you think?